marketplace/Readme.md

```diff
@@ -0,0 +1,29 @@
+# Marketplace extensions
+
+This directory contains production quality extensions for the
+[Extension Marketplace](https://www.contentful.com/developers/marketplace).
+
+These extensions are production quality and ready to use out of the box.
+
+## Development
+
+Extensions should be created using `create-contentful-extension`. After copying them over the require only minimal
+modifications:
+
+* Add a `version` property to `package.json` with the value `1.0.0` or higher.
+* Add unit tests. Preferably using Jest.
+  * Modify `.babelrc` to be environments aware
```

Comment:
Can you please add more details here?

Reply:
I added some more color. Can you look is this makes things clearer?